### PR TITLE
Enable navigation for articles & error pages

### LINKED ISSUE #3151: Add translation for `skipLinkText` used by the Navigation component
**Is your feature request related to a problem? Please describe.**
We have implemented the Navigation component on the live radio pages, however we have not add the translation `skipLinkText` for the 5 services. This is causing a propType warning and is also an a11y bug due to the link not containing any text to explain its purpose. 

image

**Describe the solution you'd like**
For the 5 services add a translation for `skipLinkText`, this can likely be copied over from the production pages but this should be confirm with Product. 

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: When running `npm run dev` ensure no propType warnings are present on the 5 services with liveradio. 

**Additional context**
We should use a screen reader to check the skip link is read out as expected when the translation is supplied.

### PR DESCRIPTION
**Pull request by sareh**
Part of #2729
Resolves #3151

**Overall change:** Enable navigation for article pagetypes. This only enables it for the services that have added navigation config in Simorgh. These are currently:

- Afaan Oromoo
- Amharic
- Igbo
- Indonesia
- Korean
- Pidgin
- Punjabi
- Thai
- Tigrinya
- Yoruba

These are identical to those on WS Front pages. There are are no navs on the other services. 

Note that there is a new toggle `navOnArticles` which is enabled locally and on `test`, but is disabled on `live`. This is because this other issue needs to be addressed before making the nav available on live articles:
https://github.com/bbc/simorgh/issues/1524

**Code changes:**

- Enable navigation for article pagetypes
- Update skip-to-content text for nav for relevant services.
- `skipLinkText` for all services that have a navigation. 
- an id on the page h1 for the skip link to anchor to. 
- cypress tests that check for a nav on the page, with a skip link and a home link. 
- Some translations for services - others will be added as they're obtained from the relevant editorial teams. 

**Testing**

Using articles available locally for these services: https://github.com/bbc/simorgh/wiki/Simorgh-Pages

Igbo 
Screen Shot Igbo article with nav
Pidgin http://localhost.bbc.com:7080/pidgin/articles/cwl08rd38l6o
Screen Shot Pidgin article with nav
Yoruba
Screen Shot Yoruba article with nav

Pidgin 404 error page with navigation
Screenshot Pidgin 404 error page with navigation

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: src/app/containers/Header/index.jsx
Comment (dr3) [inline src/app/containers/Header/index.jsx]: We can just remove this check now :P

**Review by ryanmccombe — APPROVED** [review]
Has this not broken any automated tests?  Adding a full nav bar to the page without at least having a snapshot test notice seems concerning 😅

**Comment by sareh** [comment]
@ryanmccombe Yes, I agree. 😅 Will add cypress tests for the presence of a nav bar on different services.

File: src/app/containers/Header/index.jsx
Comment (sareh) [reply src/app/containers/Header/index.jsx]: Done!

**Review by dr3 — APPROVED** [review]

**Comment by jamesbhobbs** [comment]
Aside from being massive, this looks great. What remains to put it into test?

**Comment by sareh** [comment]
@benjaminhobbs I was just getting navigation translations for services.

File: src/app/lib/config/services/korean.js
Comment (sareh) [inline src/app/lib/config/services/korean.js:37]: For some services, we do not yet have all the translations. I have added those that we have available to this PR. Once we have the remaining translations, I'll open a PR to add them.

File: cypress/integration/pages/index.js
Comment (dr3) [inline cypress/integration/pages/index.js]: Not the biggest fan of `nav a[class^="StyledLink"]` as thats down to how styled components order their class names etc, can it be done differently?

**Review by DenisHdz — APPROVED** [review]
LGTM, nice job here 👍🏻

**Review by dr3 — CHANGES_REQUESTED** [review]
The skip to content link for the article is on the H1, but in optimo land, that can be after loads of article text, so anyone who uses it might miss content. Need to discuss this before this can be merged, as IMO this isnt right

**Comment by jamesbhobbs** [comment]
> The skip to content link for the article is on the H1, but in optimo land, that can be after loads of article text, so anyone who uses it might miss content. Need to discuss this before this can be merged, as IMO this isnt right

So are you proposing that this is blocked on https://github.com/bbc/simorgh/issues/1524 @dr3 ?

**Comment by dr3** [comment]
@benjaminhobbs Unless we toggle this off on articles using a toggle, yes

**Comment by sareh** [comment]
@dr3 Have added a toggle, which sets the navigation on for Articles on local and test, but off on live. Navigation is on for all other pagetypes.

File: cypress/integration/pages/index.js
Comment (sareh) [reply cypress/integration/pages/index.js]: The class naming is due to how we've named the component itself, in conjunction with the use of styled components. These are unlikely to change, unless we have a major upgrade in styled components and even then it's unlikely for this naming convention to be broken. 

I think this is more resilient than finding the link via the DOM structure (nav > ul > li > 1).

Comment (dr3) [reply cypress/integration/pages/index.js]: tbf I prefer `nav ul li a` its clear, specific and resiliant

Comment (dr3) [reply cypress/integration/pages/index.js]: Also would `nav a` not work? What other `a` tags does the nav have?

**Review by dr3 — APPROVED** [review]

**Comment by jamesbrumpton** [comment]
lgtm

**Pull request merged**
